app/src/main/java/com/poupa/vinylmusicplayer/adapter/SongFileAdapter.java

```diff
@@ -84,7 +85,11 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int index) {
             }
         } else {
             if (holder.shortSeparator != null) {
-                holder.shortSeparator.setVisibility(View.VISIBLE);
+                if (PreferenceUtil.getInstance().useMaterialDesign()) {
+                    holder.shortSeparator.setVisibility(View.GONE);
```

Comment:
Since this code is repeated several time, please create a function somewhere that returns the View status and keep the one liner calling this new function as argument